packages/shared/styles/components/molecules/SfBanner.scss

```diff
@@ -2,80 +2,82 @@
 
 .sf-banner {
   box-sizing: border-box;
-  display: flex;
-  min-height: var(--banner-min-height, 9rem);
-  padding: var(--banner-padding, var(--spacer-big));
-  color: var(--banner-color, var(--c-text));
-  background-repeat: no-repeat;
-  background-position: var(--banner-background-position, bottom left);
-  background-size: cover;
-  background-color: var(--banner-background);
+  display: var(--banner-display, flex);
+  flex-direction: var(--banner-flex-direction, column);
+  justify-content: var(--banner-justify-content, left);
+  background-color: var(--background-color);
+  background-position: var(--banner-background-position, center);
```

Comment:
I think, we can go with `bottom left` as before,  wdyt ? @AdamPawlinski  @pspaczek

Reply:
I change to that.